client/src/components/Questions/qna.css

```diff
@@ -130,95 +133,33 @@ strong {
 }
 .answer-container {
 	width: 100%;
-	margin-bottom: 5px;
+	margin-bottom: 15px;
 	padding-left: 5px;
 }
 .a-details {
 	display: inline;
-	font-size: 12px;
-}
-
-/* Help/Report */
-.helpful-report,
-#report, #reported {
-	display: inline;
-}
-#helpful {
-	display:inline;
-	font-size: 12px;
-	border-right: 2px solid black;
-	border-left: 2px solid black;
-	padding-left: 5px;
-}
-
-#yes-btn,
-#report-btn,
-#reported {
-	font-size: 12px;
-	text-decoration: underline;
-}
-
-#reported {
-	color: red;
-	text-transform: uppercase;
-	margin-left: 10px;
-}
-
-#yes-btn:hover {
-	color: #2ACC4D;
-}
-#report-btn:hover {
-	color: red;
-}
-
-/* Answer List */
-#a-content {
-	padding-left: 5px;
+	font-size: 14px;
 }
-#a-list {
-	border-left: 3px solid #939393;
-	padding-left: 5px;
+.thumbnail-container {
+	margin-left: 20px;
 }
-#more-a, #collapse-a {
-	color: #939393;
+.thumbnail {
+	max-width: 80px;
+	max-height: 80px;
+	margin: 5px 10px 0 0;
+	padding: 3px;
 	border: 2px solid #939393;
 	border-radius: 5px;
 }
 
-#more-a:hover,
-#collapse-a:hover {
-	color: white;
-	background-color: #939393;
-}
-
-/* Answer */
-#a-name {
-	display: inline;
-	margin-left: 20px;
-}
-#a-date {
-	display: inline;
-	padding-left: 3px;
-	padding-right: 5px;
-}
-.answer-container {
-	width: 100%;
-	margin-bottom: 5px;
-	padding-left: 5px;
-}
-.a-details {
-	display: inline;
-	font-size: 12px;
-}
-
 /* Help/Report */
 .helpful-report,
 #report, #reported {
 	display: inline;
 }
 #helpful {
 	display:inline;
```

Comment:
nit: I don't think display: inline is necessary here - I believe it should default to this.